src/pint/models/jump.py

```diff
@@ -127,3 +131,10 @@ def print_par(self):
             jump_par = getattr(self, jump)
             result += jump_par.as_parfile_line()
         return result
+
+    def get_number_of_jumps(self):
```

Comment:
@luojing1211 can you review the changes in this file and confirm that you agree  with them?